src/js/common/schemaform/helpers.js

```diff
@@ -141,7 +141,7 @@ export function formatReviewDate(dateString) {
   return undefined;
 }
 export function parseISODate(dateString) {
-  if (dateString) {
+  if (typeof dateString === 'string') {
```

Comment:
Oh, this was back when the DefinitionTester passed non-objects as data. An object was passed to this function which, of course, passed this statement, but failed when we tried to `dateString.split()`.

Shouldn't be a problem now, but it just stays there as a sort of safeguard.